src/spellcheckercoresettings.cpp

```diff
@@ -95,6 +98,7 @@ bool SpellCheckerCoreSettings::operator==( const SpellCheckerCoreSettings& other
   different = different | ( checkExternalFiles != other.checkExternalFiles );
   different = different | ( projectsToIgnore != other.projectsToIgnore );
   different = different | ( replaceAllFromRightClick != other.replaceAllFromRightClick );
+  different |= errorsColor != other.errorsColor;
```

Comment:
Please stick with the pattern used by the other checks.

🤔 What is the minimum compiler version for Qt Creator at this time, can't this whole function be replaced with a default comparison operator yet?